packages/mc-scripts/src/commands/build.ts

```diff
@@ -44,16 +44,7 @@ async function run() {
     if (result.warnings.length) {
       console.log(chalk.yellow('Compiled with warnings.\n'));
       console.log(result.warnings.join('\n\n'));
-      console.log(
-        `\nSearch for the ${chalk.underline(
-          chalk.yellow('keywords')
-        )} to learn more about each warning.`
-      );
-      console.log(
-        `To ignore, add ${chalk.cyan(
-          '// eslint-disable-next-line'
-        )} to the line before.\n`
-      );
```

Comment:
we don't run eslint during our build, no need to special handle eslint warnings like this.